vic/drivers/image/Makefile

```diff
@@ -75,6 +75,7 @@ INCLUDES = -I ${DRIVERPATH}/include \
 
 # Uncomment to include debugging information
 CFLAGS  =  ${INCLUDES} ${NC_CFLAGS}  -ggdb -O0 -Wall -Wextra -std=c99 \
+					 -fopenmp \
```

Comment:
Right, we just need to make sure we pass the `-openmp` (or equivalent) flag to the compiler for VIC. That will need to be done in RASM repo, not the VIC repo.